docs/UserGuide.md

```diff
@@ -199,6 +244,13 @@ Exits the program.
 
 Format: `exit`
 
+<div markdown="span" class="alert alert-primary">
+  :bulb: <b>Tip:</b>
+  As mentioned in the <code>view</code>, <code>exit</code> exits the program when executed on normal foster family list view, while on the profile view page you are exited out of the page back to the list view. 
```

Comment:
I think can add a "the" before "normal foster family list view" and a  comma after "while on the profile view page"